.github/CODEOWNERS

```diff
@@ -1095,6 +1095,7 @@ x-pack/test_serverless/api_integration/test_suites/common/platform_security @ela
 src/plugins/discover/public/context_awareness/profile_providers/security @elastic/kibana-data-discovery @elastic/security-threat-hunting-investigations
 
 # Platform Docs
+/x-pack/test/functional/services/sample_data @elastic/platform-docs
```

Comment:
The sample data functionality is implemented in the [home plugin](https://github.com/elastic/kibana/tree/main/src/plugins/home) which is owned by the @elastic/kibana-core team. So I think it would make sense that this teams also owns the corresponding test helpers that are using the APIs provided by that plugin, following our principle that tests / test helpers should be close to the functionality they testing, which starts with code ownership.

Reply:
@wayneseymour , my apologies, I believe we need to update this to @elastic/appex-sharedux ... I see we are listed as the owners in `home/kibana.jsonc` but I think this is no longer maintained by our team.

Reply:
Does that mean that `home/kibana.jsonc` should be updated to relfect this as well?  It seems so to me.

Reply:
fyi: [follow-up / fix pr](https://github.com/elastic/kibana/pull/200564)